lightly/api/api_workflow_predictions.py

```diff
@@ -13,14 +13,14 @@ def create_or_update_prediction_task_schema(
         schema: PredictionTaskSchema,
         prediction_version_id: int = -1,
     ) -> None:
-        """Creates or updates the prediction task schema
+        """Creates or updates the prediction task schema.
 
         Args:
             schema:
                 The prediction task schema.
             prediction_version_id:
-                A numerical id (e.g timestamp) to distinguish different predictions of different model versions.
-                Use the same id if you don't require versioning or if you wish to overwrite the previous schema.
+                A numerical ID (e.g., timestamp) to distinguish different predictions of different model versions.
```

Comment:
Ary you sure the extra `,` is needed here?

Reply:
Mostly yes for American English.